pom.xml

```diff
@@ -137,7 +137,7 @@
     <findbugs.skip>false</findbugs.skip>
     <grpc.version>1.54.1</grpc.version>
     <gson.version>2.8.9</gson.version>
-    <guava.version>31.0.1-jre</guava.version>
+    <guava.version>32.0.0-jre</guava.version>
```

Comment:
according to the guava release notes https://github.com/google/guava/releases/tag/v32.0.1 and the linked CVE description, this upgrade should be to version `32.0.1`